src/components/UserInfo/UserInfo.tsx

```diff
@@ -2,0 +1,21 @@
+import React from 'react';
+import { Todo } from '../../types/types';
+import { getUser } from '../../utils/getUser';
+
+type Props = {
+  todo: Todo;
+};
+
+export const UserInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUser(todo.userId);
```

Comment:
The `getUser` function should not be called here. Instead, pass the user data directly to this component as a prop from the parent component.